From: uazo <uazo@users.noreply.github.com>
Date: Wed, 22 May 2024 07:43:04 +0000
Subject: Keep disabled WebViewMediaIntegrityApi

Disables a new mechanism similar to Web Environment Integrity API in use in webviews
see https://issuetracker.google.com/issues/327186031
https://android-developers.googleblog.com/2023/11/increasing-trust-for-embedded-media.html

need: Add-cromite-flags-support.patch
License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 android_webview/common/aw_features.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android_webview/common/aw_features.cc b/android_webview/common/aw_features.cc
--- a/android_webview/common/aw_features.cc
+++ b/android_webview/common/aw_features.cc
@@ -73,6 +73,7 @@ BASE_FEATURE(kWebViewLazyFetchHandWritingIcon,
 BASE_FEATURE(kWebViewMediaIntegrityApiBlinkExtension,
              "WebViewMediaIntegrityApiBlinkExtension",
              base::FEATURE_ENABLED_BY_DEFAULT);
+SET_CROMITE_FEATURE_DISABLED(kWebViewMediaIntegrityApiBlinkExtension);
 
 // When enabled, passive mixed content (Audio/Video/Image subresources loaded
 // over HTTP on HTTPS sites) will be autoupgraded to HTTPS, and the load will be
--
